sub-packages/bionemo-scdl/tests/bionemo/scdl/conftest.py

```diff
@@ -29,7 +29,8 @@ def test_directory() -> Path:
     Returns:
         A Path object that is the directory with test data.
     """
-    return load("scdl/sample") / "scdl_data"
+    # return load("scdl/sample") / "scdl_data"
+    return load("scdl/sample_scdl_feature_ids", source="pbss") / "scdl_data_with_feature_ids"
```

Comment:
Do all the rest of our tests still use pbss as the data source? There was a recent push to move everything to public NGC.